R/database.R

```diff
@@ -16,6 +16,14 @@ maf_header = c("Hugo_Symbol"=1,"Entrez_Gene_Id"=2,"Center"=3,"NCBI_Build"=4,"Chr
 #'
 get_excluded_samples = function(tool_name = "slms-3"){
   base = config::get("repo_base")
+
+  #check for missingness
+  path = paste0(base,"config/exclude.tsv")
+  if(!file.exists(path)){
```

Comment:
This is looking in the gambl repo rather than for synced data. Be careful your error messages are not misleading. This should say something along the lines of "have you cloned the GAMBL repo and added the path to this directory under the local section of  your config?" Ideally we would point to some documentation here too.

Reply:
Easiest way to work out which is which is that there are two base paths:
repo_base (path to your GAMBL repo)
project_base (where the data will actually be sitting)